src/Mizar/SamplingWithFrameTrackInputWidget.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_SAMPLING_WITH_FRAME_TRACK_INPUT_WIDGET_H_
```

Comment:
I think both Widget classes should be moved into a separate module - maybe `MizarWidgets` or something else to your liking. The reason is that otherwise they can't be tested.